src/react/src/features/router/Router.tsx

```diff
@@ -23,23 +23,25 @@ export function Router() {
         dispatch(handleSelect(val));
     };
     //@todo need to add a tab for account
+    //@todo redirect back to login if a user isn't logged in
     return (
         <BrowserRouter basename='/'>
-            <Tabs value={tabValue} onChange={handleTabs} centered>
-                {tabComponent}
-            </Tabs>
             <Switch>
-                <Route exact path="/">
-                    <Redirect to="/collection" />
+                <Route exact path='/' component={Login} />
+                <Route exact path='/register' component={Register} />
+                <Route>
+                    <Tabs value={tabValue} onChange={handleTabs} centered>
```

Comment:
had to do some rearranging so that tabs wouldn't show up on login/register

Reply:
So I think routes need to be tied to a path not wrap the elements

Reply:
it seems to be working fine so far